apps/docs/content/components/Card/index.mdx

```diff
@@ -0,0 +1,102 @@
+
+### Heading
+
+The heading is the main title of the card. It should be short and concise.
+
+### Description
+
+The description is an optional short text that extends the information provided by the heading.
+
+### Call to action
+
+The call to action label indicates the action or resource the card links to. It defaults to "Learn more".
+
+## Usage
+
+Use cards to summarize the core idea behind some content and link to its detailed resource. Always display information in a compact way and avoid long texts or complex information.
+
+Cards should always be used in groups of two or more related concepts.
+
+### Stacked
+
+Cards should be stacked to display a list of items that are related to each other. If displaying a visual asset, do so for all cards to achieve a visually balanced layout. The visual asset should be the same type for all cards, either an icon or an image.
+
+For a better visual experience, we recommend using similar length size for the heading and description in all stacked cards.
+
+<DoDontContainer stacked>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/912236/66394dda-13de-4c86-bd82-534d99d593d8" />
+    <Caption>
+      Use the same type of visual asset for all cards and a similar lenght size
+      for the content.
+    </Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/912236/c612ccf8-7c98-4213-92b9-2fdf6f0f1398" />
+    <Caption>
+      Don't use different lenght sizes for the title or the description.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+When using an image as the visual asset, we recommend using the same aspect ratio for all images. This will create a more balanced visual experience.
+
+<DoDontContainer stacked>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/912236/0ec93c16-7091-4ebb-a1a0-b0626de7833f" />
+    <Caption>
+      Use the same image ratio and a similar lenght sizes for the content.
+    </Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/912236/0b8ffb2d-0753-4bdd-9165-0f30ec28a8c8" />
+    <Caption>
+      Don't use different image ratios and a different length size for the
+      content.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+Cards typically appear in groups of three or four. For rows of four cards, we recommend using short and concise content and avoid using images as the visual asset. When more than four cards or extensive content is needed, stack them vertically.
+
+<DoDontContainer stacked>
+  <Do>
+    <img src="https://github.com/primer/brand/assets/912236/a56f80f9-9a02-4fe0-8d8b-eb61e8396030" />
+    <Caption>
+      Stack cards vertically using multiple rows so that the content is easy to
+      scan.
+    </Caption>
+  </Do>
+  <Dont>
+    <img src="https://github.com/primer/brand/assets/912236/3f467a76-da93-4b1c-a53e-763319df8049" />
+    <Caption>
+      Don't use a single row of cards if the content is too extensive.
```

Comment:
I think this would happen even if content was concise. Should the advise here be to stack vertically on narrower viewports instead?

Reply:
👍  I'll add that to the description and examples

Reply:
Updated the description with a video player with a responsive example

https://github.com/primer/brand/assets/912236/e4ac4ed3-4190-4e8b-b467-364fe880aa30